watertap/unit_models/electrodialysis_0D.py

```diff
@@ -1546,6 +1716,179 @@ def eq_dl_thickness_ioa(self, mem, side, t):
                     * self.current_dens_lim_ioa[t] ** -1
                 )
 
+    def _get_fluid_dimensionless_quantities(self):
+        self.diffus_mass = Var(
+            initialize=1e-9,
+            bounds=(1e-16, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="The mass diffusivity of the solute as molecules (not individual ions)",
+        )
+        self.hydraulic_diameter = Var(initialize=1e-3, units=pyunits.meter)
```

Comment:
Why are you omitting the full configuration of bounds and doc here? Probably worth including

Reply:
Thanks for the suggestion, the bounds and docs were added to both 0D and 1D.